contrib/epee/include/misc_language.h

```diff
@@ -95,16 +97,7 @@ namespace misc_utils
       return memcmp(&_Left, &_Right, sizeof(_Left)) < 0;
   }
 	
-
-	inline
```

Comment:
This seems unrelated to the other changes, but it is small.

Reply:
It prevents propagation of boost::thread. It appears small, but can make a large difference cumulative.